design-patterns/structural-patterns/src/facade/Address.java

```diff
@@ -0,0 +1,32 @@
+package facade;
+
+public class Address {
+	
+	private String id;
+	private String streetName;
+	private String city;
+	
+	public String getId() {
+		return id;
+	}
+
+	public void setId(String id) {
+		this.id = id;
+	}
+	
+	public String getStreetName() {
+		return streetName;
+	}
+	
+	public void setStreetName(String streetName) {
```

Comment:
I would suggest that to use the constructor instead of setters. Use setters when you need to modify the object in tun time. currently you are using these setters only to create a new address.

Reply:
Fixed.